bvm/ballerina-core/src/main/java/org/ballerinalang/util/codegen/Mnemonics.java

```diff
@@ -79,6 +79,7 @@
         mnemonics[InstructionCodes.FREEZE] = "freeze";
         mnemonics[InstructionCodes.IS_FROZEN] = "isFrozen";
 
+        mnemonics[InstructionCodes.STAMP] = "stamp";
```

Comment:
This needs to be moved before `mnemonics[InstructionCodes.FREEZE] = "freeze";` right?

Reply:
Same with the other too.